src/main/clojure/clara/rules/engine.cljc

```diff
@@ -118,6 +118,14 @@
   ;; Right-activate the node with items reduced in the above pre-reduce step.
   (right-activate-reduced [node join-bindings reduced  memory transport listener]))
 
+(defprotocol IAccumInspect
+  ;; Takes a token that was previously propagated from the node
```

Comment:
Also, am I correct in understanding this accepts a token from this node, or a descendent of that token, and returns the corresponding elements? It might be worth making it more explicit what exactly the input and output of this would be.